isis/src/osirisrex/apps/ocams2isis/ocams2isis.xml

```diff
@@ -10,6 +10,16 @@
       This program will import FITS (Flexible Image Transport System) formatted files from the
       OSIRIS-REx MapCam, SamCam, and PolyCam instruments into the ISIS format.
     </p>
+    <p>
+      Current known errors from OREx IPWG work that have been fixed in their version but not in the public version:
+      <ul>
+        <li>Focal position dependent distortion for the PolyCam model is not implemented</li>
+        <li>The current distortion is out of date, software changes are needed to use the OpenCV based distortion in the latest IKs</li>
+        <li>The Struct SPK is not being used which accounts for the positions of the instruments relative to the center of the spacecraft. From the struct kernel docs this will result in an error of about a meter.</li>
+        <li>Only specific sites have DSKs available. Software changes are required to load multiple DSKs at the same time. It is recommended that you only work within the extents of the DSK you are using. Use the dskbrief utility to view the lat/lon extents of your DSK file.</li>
+      </ul>
+      Based on all of these and what the IPWG team found, it is recommended that users don't attempt to do sub-meter processing until we can get the IPWG's fixes integrated.
```

Comment:
I would change this last part to:

"Based on these issues, it is recommended that users don't attempt to do sub-meter or color processing with this version of the software."

We can update this warning as things get fixed.